### PR TITLE
Add transfer endpoint to kvstore

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

**Review by fmorency — COMMENTED** [review]
The code LGTM, but I'm wondering why you need two address parameters. 

See https://github.com/many-protocol/specification/pull/86#discussion_r1053716809

File: src/many-modules/src/_4_events.rs [hunk 1]
```diff
@@ -659,6 +659,11 @@ define_event! {
         2     | owner:                  Option<Address>                        [ id_non_null ],
         3     | reason:                 Option<Reason<u64>> ,
     },
+    [7, 2]      KvStoreTransfer (module::kvstore::TransferArgs) {
+        1     | key:                    ByteVec,
+        2     | owner:                  Option<Address>                        [ id_non_null ],
```
Comment (fmorency) [inline src/many-modules/src/_4_events.rs]: `owner` should not be an `Option`

**Review by fmorency — CHANGES_REQUESTED** [review]
I can't comment directly in the code so here it is.

- `owner` in `KvStorePut` event should not be an `Option`
- `owner` in `KvStoreDisable` event should be removed. 
- `reason` in `KvStoreDisable` should have id `2` and should not be an `Option`

You should add the new `canKvStoreTransfer` role in `_9_account.rs` and `_9_account/features/kvstore.rs`

File: src/many-modules/src/_4_events.rs (hunk 1 above)
Comment (hansl) [reply src/many-modules/src/_4_events.rs]: Did you review the right file/commit?

**Review by fmorency — APPROVED** [review]
LGTM assuming CI passes

**Pull request merged**